lib/postzord/receiver/local_batch.rb

```diff
@@ -61,5 +61,8 @@ def notify_users
     @users.find_each do |user|
       Notification.notify(user, @object, @object.author)
     end
+    if @object.target.author.owner
```

Comment:
Use a guard clause instead of wrapping the code inside a conditional expression.